rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -46,13 +46,24 @@ object RhoType {
       p.singleExpr().collect {
         case Expr(GByteArray(bs)) => bs.toByteArray
       }
+
+    def apply(bytes : Array[Byte]) : Par =
+      Expr(GByteArray(ByteString.copyFrom(bytes)))
   }
 
   object String {
     def unapply(p: Par): Option[String] =
       p.singleExpr().collect {
         case Expr(GString(bs)) => bs
       }
+
+    def apply(s : String) : Par = GString(s)
+    def apply(maybeStr : Option[String]) : Par =
```

Comment:
please get rid of this - I'd rather have this Option handled explicitly

Reply:
True.

It seemed like a good idea at the time.